src/EnergyPlus/SZVAVModel.cc

```diff
@@ -1131,15 +1131,16 @@ namespace SZVAVModel {
             coilActive = DataLoopNode::Node(coilAirInletNode).Temp - DataLoopNode::Node(coilAirOutletNode).Temp;
 
             if (!coilActive) { // if the coil is schedule off or the plant cannot provide water
-                DataLoopNode::Node(coilFluidInletNode).MassFlowRate = 0.0;
-                if (coilLoopNum > 0)
+                if (coilLoopNum > 0) {
```

Comment:
I wasn't really expecting this change, but it's reasonable to protect the node mass flow rate evaluation to only cases where the coil has a loop number.